source/includes/_email_survey_csv.md

```diff
@@ -0,0 +1,172 @@
+#Email Survey CSV
+
+```sh
+// Basic Example
+
+curl "https://api.wootric.com/v1/email_survey_csv" \
+  -d "access_token=XXXXXXXXXXXXXXXX" \
+  -d "csv=FILE.csv"
+  -d "survey_immediately=true" \
+  -d "survey_settings[custom_messages][followup_text]=Thank you!" \
+  -d "subject=Would you mind sharing your thoughts about our service?" \
+  -d "intro=We want you to be our customer for life. That's why your opinion matters."
+```
+
+CSV file 
+
+email,segment1,segment2
+email@email.com,segment1_value,segment2_value
+email2@email.com,2segment1_value,2segment2_value
+email3@email.com,3segment1_value,3segment2_value
+email4@email.com,4segment1_value,4segment2_value
+
+- Maximum of 5000 rows per request.
+- We will automatically create an end user if they don't exist. ???
+
+Param | Type | Description
+----- | ---- | ------------
+csv   | File | CSV file with end user data
+subject | String | Override default subject line (standard NPS/CES/CSAT question used by default). Can handle placeholder values surrounded by {{my_value}}
+intro | String | Intro message for your email. Displayed between logo and question. Default: none. Can handle placeholder values surrounded by {{my_value}}
+context | Hash Array | Array of hashes containing the values to be replaced in subject and/or intro
+senders_name | String | Override senders name
+reply_to_email | String | Override reply-to email
+delay | Integer | Days we will wait before sending the survey. Delay is optional and it must be between 1 and 31 days.
+delivery_time | String | UTC time of the day to deliver emails (format: hh:mm) in 24 hs format.
+custom_sender_email | String | Custom sender email.
+send_to_email_field | String | Column containing the email address of the receiver.
+unique_identifier_field | String | Field used to set the email of the end user (require send_to_email_field).
+survey_immediately | Boolean | Override eligibility check
+survey_settings | Hash | See **survey_settings** parameters below
+
+
```

Comment:
You may also want to add a small description of the possible CSV columns, the first column *must* be the `email` and then we have special columns like `___context:var` and  `___language`, stating that any other column would be treated as a string property (columns ending with _amount would be Integers and if they end with _date they will be treated as dates)